dockernet/src/init_chain.sh

```diff
@@ -139,12 +137,14 @@ for (( i=1; i <= $NUM_NODES; i++ )); do
     # Add this account to the current node
     $cmd add-genesis-account ${val_addr} ${VAL_TOKENS}${DENOM}
 
-    if [[ $CHAIN != "STRIDE" ]]; then
-        cp $DOCKERNET_HOME/state/${STRIDE_NODE_PREFIX}${i}/config/priv_validator_key.json $DOCKERNET_HOME/state/${NODE_PREFIX}${i}/config/priv_validator_key.json
-        cp $DOCKERNET_HOME/state/${STRIDE_NODE_PREFIX}${i}/config/node_key.json $DOCKERNET_HOME/state/${NODE_PREFIX}${i}/config/node_key.json
+    if [[ $CHAIN == "GAIA" ]]; then
+        stride_config=$DOCKERNET_HOME/state/${STRIDE_NODE_PREFIX}${i}/config
+        host_config=$DOCKERNET_HOME/state/${NODE_PREFIX}${i}/config
+        cp ${stride_config}/priv_validator_key.json ${host_config}/priv_validator_key.json
+        cp ${stride_config}/node_key.json ${host_config}/node_key.json
     fi
 
-    if [[ ($CHAIN == "STRIDE" && ($i == 1 || $i == 2)) || $CHAIN != "STRIDE" ]]; then
+    if [[ ($CHAIN == "STRIDE" && ($i != 0)) || $CHAIN != "STRIDE" ]]; then
```

Comment:
The index `i` of this loop actually starts from 1. 
```suggestion
    if [[ ($CHAIN == "STRIDE" && ($i <= 2)) || $CHAIN != "STRIDE" ]]; then
```

Reply:
chatted offline, decided to remove this if statement